pytket/pytket/qasm/qasm.py

```diff
@@ -1054,6 +1071,11 @@ def circuit_to_qasm_io(
                 if param > 1:
                     param = -2 + param
             params = [param]
+        elif optype == OpType.Barrier:
```

Comment:
Barriers are already handled in the normal case, so this should only apply if the header is `hqslib1_dev`.

Reply:
done